core/solidus_core.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'ffaker', '~> 2.0'
   s.add_dependency 'friendly_id', '~> 5.0'
   s.add_dependency 'highline', '~> 1.7' # Necessary for the install generator
-  s.add_dependency 'kaminari', '~> 0.15', '>= 0.15.1'
+  s.add_dependency 'kaminari', '~> 1.0'
```

Comment:
Sure, that makes sense, I'll change it to be that.  (I'm not sure it actually has any intentional breaking changes, I couldn't find any; but may have unintentional breaking changes and bugs; and I agree with the principle to not force people to upgrade)

This _was_ allowing as low as `0.15.1` before, does that not work on Rails5 at all, I should require at least